pyccl/halos/halo_model.py

```diff
@@ -509,6 +510,18 @@ def halomod_power_spectrum(cosmo, hmc, k, a, prof,
             term in the first equation above) won't be computed.
         get_2h (bool): if `False`, the 2-halo term (i.e. the second
             term in the first equation above) won't be computed.
+        smooth_transition (function or None):
+            Modify the halo model 1-halo/2-halo transition region
+            via a time-dependent function :math:`\\alpha(a)`,
+            defined as in HMCODE-2020: :math:`P(k,a)=
```

Comment:
Can you add a reference to the paper (either arXiv reference or Mead et al. 2020)

Reply:
Done.